runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotConfig.java

```diff
@@ -124,19 +124,41 @@ default ImageUploadStyle uploadScreenshot()
 	@ConfigItem(
 		keyName = "kills",
 		name = "Screenshot PvP Kills",
-		description = "Configures whether or not screenshots are automatically taken of PvP kills",
+		description = "Configures whether or not screenshots are automatically taken of PvP kills. This does not include kills in LMS or Tournament worlds",
 		position = 8
 	)
 	default boolean screenshotKills()
 	{
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "lmskills",
+			name = "Screenshot LMS Kills",
+			description = "Configures whether or not screenshots are automatically taken of LMS kills",
+			position = 9
```

Comment:
Your auto-formatter indented this one level too far. Make sure both added configs only add a single level of indent for the `@ConfigItem` annotations.